Sources/Jetworking/Client/Client.swift

```diff
@@ -356,7 +356,10 @@ extension Client: DownloadExecutorDelegate {
 
     public func downloadExecutor(_ downloadTask: URLSessionDownloadTask, didCompleteWithError error: Error?) {
```

Comment:
Maybe we should rename the delegate function to `didFailWithError error: Error`? What do you think?

Reply:
I don't think so. This is basically the same naming as in the `URLSessionDownloadDelegate`. We just "convert" two delegate methods (didCompleteWithError and didFinishDownloadingTo)  into one completion. And therefore we should keep track when it's appropriate to call the completion.

Reply:
Hm ok, then we could leave it as is. Can you please add documentation or rethink the visibility of the function? Is it necessary, that it is available from outside of the framework?